lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -333,16 +333,24 @@ void SessionState::sess_infocopy(struct SessionInfo* info) {
   // Static SessionInfo vlaue till UPF node value implementation
   // gets stablized.
   // TODO we cud eventually  migrate to SMF-UPF proto enum directly.
-  info->state = get_proto_fsm_state();
-  info->subscriber_id.assign(imsi_);
-  info->ver_no              = get_current_version();
-  info->local_f_teid        = get_upf_local_teid();
-  info->nodeId.node_id_type = SessionInfo::IPv4;
-  info->pdr_rules           = get_all_pdr_rules();
-  if (!info->pdr_rules.empty()) {
-    // Get the UE ip address from first rule
-    auto& rule    = info->pdr_rules.front();
-    info->ip_addr = rule.pdi().ue_ipv4();
+  if (info != nullptr) {
```

Comment:
When do we get a nullptr into this function? Is it OK that we just silently ignore the nullptr and do nothing here? And if that's the case, let's check for null first and return early to clean it up.

Reply:
Modified the code so that it returns early if nullptr found, as you suggested.